public/content/documentation/web/component/link.md

```diff
@@ -6,75 +6,485 @@ How to test a link
 
 2. Test mobile screenreader gestures
 
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+   - Swipe: Focus moves to the link
+   - Doubletap: Activates the link
 
 3. Listen to screenreader output on all devices
 
-   - It is discoverable with screenreader shortcuts as header/banner landmark
+   - Name: Its purpose is clear
+   - Role: It identifies itself as a link
 
-   - Group: It typically contains the name and primary navigation of the website
 
 Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link)
 
 ## Gherkin
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a link
 
-GIVEN THAT I am on a page with a header landmark
+GIVEN THAT I am on a page with a link
 
 1. Keyboard for mobile & desktop
 
-   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+   - WHEN I use the tab key to move focus to a link I SEE focus is strongly visually indicated
+   - THEN when I use the enter key to activate the link I SEE my browser goes somewhere
 
 2. Desktop screenreader
 
-   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND 
+   - I use the tab key to move focus to a link
+      - I HEAR Its purpose is clear
+      - I HEAR It identifies itself as a link
+   - THEN when I use the enter key to activate the link I HEAR my browser goes somewhere
 
-   - I use the tab key to enter the web browser window
+3. Mobile screenreader
 
-   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+   - I swipe to focus on a link
+      - I HEAR Its purpose is clear
+      - I HEAR It identifies itself as a link
+   - THEN when I doubletap with the link in focus I HEAR my browser goes somewhere
 
-   - I HEAR It typically contains the name and primary navigation of the website
 
-3. Mobile screenreader
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link)
 
-   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
 
-   - I swipe to focusable elements in the header
+## Links vs buttons
 
-   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+### If it goes somewhere, it's `<a>` link.
 
-   - I HEAR It typically contains the name and primary navigation of the website
+- When the user clicks a link, they are taken to a different location in the site.
+  - Either another page or even another area of the same page
+- A link can look like a big shiny button but it must be coded as `<a>` link
 
+### If it does something, it's a `<button>`
```

Comment:
Similar to the first comment. Up to you for consistency, either:
a. add a period at the end of this heading ("If it does something, it's a <button>") OR
b. remove the period at the end of the previous ("If it goes somewhere, its <a> link. ") heading.